sessionctx/variable/session.go

```diff
@@ -1033,6 +1033,10 @@ type SessionVars struct {
 
 	// MetricSchemaStep indicates the step when query metric schema.
 	MetricSchemaStep int64
+
+	// TiDBWriteByTiCDC indicates whether the following data is written by TiCDC.
+	TiDBWriteByTiCDC bool
```

Comment:
```suggestion
	// WriteByTiCDC indicates whether the following data is written by TiCDC.
	WriteByTiCDC bool
```

Please keep consistent with other variables.